src/transactions.ts

```diff
@@ -6,6 +6,7 @@ const DEFAULT_GOVERNOR_PROXY = "0x2d56dC077072B53571b8252008C60e945108c75a";
 
 export const loadProposal = async (num: string) => {
   const x = await import(`../vips/vip-${num}.ts`);
+  num = num.replace("-testnet", "Testnet");
```

Comment:
I think I wouldn't force us to do this. I would even stop using the prefix "Testnet" in the name of the functions. The functions are in different files, so we can use `vip152()` in both cases

Reply:
Let's keep this change, this way we can use a flat file structure from now on